test/testdata.js

```diff
@@ -86,6 +86,8 @@ var payProData = {
   toAddress: 'mjfjcbuYwBUdEyq2m7AezjCAR4etUBqyiE',
   amount: 404500,
   network: 'testnet',
+  outs:
+   [ { addr: 'mjfjcbuYwBUdEyq2m7AezjCAR4etUBqyiE', amount: 404500 } ],
```

Comment:
Please add 1 test with more than 1 output.

Reply:
These tests are old and defunct. They were put in place by Bitpay, who had uploaded binary files matching the data in testdata. There are no binary files with multiple outputs to match against. I would need to redo basically all of the tests to do this.